server/cmd/ptrun-server/containertest.go

```diff
@@ -0,0 +1,124 @@
+package main
```

Comment:
Ta datoteka je primer, ki sem ga uporabljal pri Dockerju, ker vsebuje tako HTTP URLje kot tudi WebSocket, in to v istem threadu...